web/src/lib/components/elements/buttons/circle-icon-button.svelte

```diff
@@ -1,8 +1,8 @@
 <script lang="ts">
   import type Icon from 'svelte-material-icons/AbTesting.svelte';
 
+  export const backgroundColor = 'transparent';
```

Comment:
I think if you use `export const`, other parent component cannot change the props value of this button. You should use `export let` instead

Reply:
It sounds like you are just not using the property anymore. You could leave it as it was before, but change the default value instead.

Or, if you are going to remove it, you will need to see if it is used anywhere and if this change breaks anything for those uses.

Reply:
I see that `backgroundColor` is used all over the place. Therefore removing it was not a wise choice cause it might cause side effects :smiling_face_with_tear:  . What if I set the `style:background-color={backgroundColor}` only if `backgroundcolor` is explicitly set?

Reply:
> I see that `backgroundColor` is used all over the place. Therefore removing it was not a wise choice cause it might cause side effects 🥲 . What if I set the `style:background-color={backgroundColor}` only if `backgroundColor` is explicitly set?

Yeah this should work